eng/pipelines/stress-http.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
```

Comment:
could you also disable PR triggers for now? I guess we don't want to run this build in PRs just yet, right?

Reply:
Aren't those opt-in? We never want to trigger on PRs for stress.

Reply:
But do you ever want to trigger them via comments, as an optional build?

Reply:
Yes, this is something to iterate on.

Reply:
Ok, if so, you will need to define PR triggers here and then on the UI you'll have to set it up accordingly.

Reply:
Yup